squareHodoScopeSimulation

```diff
@@ -0,0 +1,172 @@
+import random
+import math
+import numpy as np
+import ROOT
+ROOT.gROOT.SetBatch(True)
+
+def makeBranch(tree, name, t):
+    b = None
+    if t == "vector":
+        b = ROOT.vector('double')()
+        tree.Branch(name, b)
+    elif t == "D":
+        b = np.zeros(1, dtype=float)
+        tree.Branch(name, b, name+"/"+t)
+    else:
+        print("branch type error")
+    return b
+
+def saveHisto(c, hDic, name, drawOpt):
```

Comment:
Let's remove the histogram making stuff from this file. Have it only in the Reco file.